stripper/ze_ffxii_westersand_v8/default_ents.jsonc

```diff
@@ -24,6 +24,18 @@
 						"overrideparam": "mp_flashlight 1"
 					}
 				]
+			},
+			// Kill epick blockers that were added in port
+			"insert": {
+				"io": [
+					{
+						"outputname": "OnMapSpawn",
+						"targetname": "Staff_*_Epicker",
+						"inputname": "Kill",
+						"delay": 10.0,
+						"timestofire": 1
+					}
+				]
 			}
```

Comment:
Can you move this to a new root modify block? It's just a bit weird having this mixed in with another change.

Also, I assume you went with an input rather than `filter` just to avoid having to stripper a bunch of different template lumps? Seems more efficient.

Reply:
I'll move to a different modify block. Just not used to the format so taking some time to get used to. And yes, I put it as an input because manually going through every single lump and removing the filter would take a lot of time.